src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.figure.enums.Color;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
```

Comment:
for each type of figure

```suggestion
    public Circle(int radius, String color) {
```